backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ProjectDatabaseHandler.java

```diff
@@ -1145,9 +1181,15 @@ private String extractReleaseNameForClearingEmail(List<Release> releases) {
         return builder.toString();
     }
 
-    private int getTotalReleaseCount(ReleaseClearingStateSummary clearingSummary) {
-        return clearingSummary.getNewRelease() + clearingSummary.getReportAvailable() + clearingSummary.getUnderClearing()
-                + clearingSummary.getSentToClearingTool()+ clearingSummary.getApproved();
+    private String appendCompTypeToReleaseVersion(Collection<Release> releases, Set<String> cotsCompIds) {
+        Integer count = 0;
```

Comment:
Use `int` to prevent `Autoboxing` and `Unboxing` during increment and then convert it to string.